app/lib/pre_assembly/digital_object.rb

```diff
@@ -73,6 +74,12 @@ def pre_assemble
                  message: "can't be opened for a new version; cannot re-accession when version > 1 unless object can be opened" }
       end
 
+      if ObjectFileValidator.new(object: self, batch: batch).object_has_hierarchy? && content_structure != 'file'
+        return { pre_assem_finished: false,
+                 status: 'error',
+                 message: "can't be accessioned -- if object files have hierarchy the content structure must be set to file" }
+      end
+
       @assembly_directory = AssemblyDirectory.create(druid_id: druid.id, base_path: container)
```

Comment:
this is so that any individual object in a pre-assembly job will not be continue if the object is found to have hierarchy and the content_structure is not file